data/sles4sap/b1_installer.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+install_bin="$1"
+b1_cfg="$2"
+
```

Comment:
What about adding a comment in this script explaining what it is about and (or maybe even only) why it is needed.